L1Trigger/TrackFindingTracklet/interface/CircularBuffer.h

```diff
@@ -0,0 +1,63 @@
+#ifndef CIRCULARBUFFER_H
+#define CIRCULARBUFFER_H
+
+#include <assert.h>
+#include <vector>
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files